steps/olm/operator.go

```diff
@@ -160,3 +160,39 @@ var _ = gauge.Step("Verify <resourceType> Results records", func(resourceType st
 var _ = gauge.Step("Verify <resourceType> Results logs", func(resourceType string) {
 	operator.VerifyResultsLogs(resourceType)
 })
+
+var _ = gauge.Step("Verify <resourceType> signature", func(resourceType string){
+	operator.VerifySignature(resourceType)
+})
+
+var _ = gauge.Step("Start task", func(){
+	operator.StartKanikoTask()
+})
+
+var _ = gauge.Step("Verify image signature", func(){
+	operator.VerifyImageSignature()
+})
+
+var _ = gauge.Step("Check Attestation", func(){
+	operator.CheckAttestation()
+})
+
+var _ = gauge.Step("Verify Attestation", func(){
+	operator.VerifyAttestation()
+})
+
+var _ = gauge.Step("Import image registry variables", func(){
+	if os.Getenv("CHAINS_REGISTRY") == "" || os.Getenv("CHAINS_REPOSITORY") == "" {
+		testsuit.T.Errorf("'CHAINS_REGISTRY' and 'CHAINS_REPOSITORY' image registry system variables were not exported")
+	}else{
+		fmt.Println("Image registry variables were succesfully imported")
+	}
+})
+
+var _ = gauge.Step("Create signing-secrets for tekton chains", func(){
+	if oc.SecretExists("signing-secrets", "openshift-pipelines"){
```

Comment:
Empty signing secrets exists after installation, you need to have a peek on content of the secret